lib/routes/cqu/jwc/jwc.ts

```diff
@@ -0,0 +1,75 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/jwc/:path{.+}?',
+    categories: ['university'],
+    example: '/cqu/jwc/index/tzgg',
+    parameters: {
+        path: {
+            description: '路径参数，默认为 `index/tzgg`',
+            default: 'index/tzgg',
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+        source: ['jwc.cqu.edu.cn/:path'],
+        target: '/jwc/:path',
+        }
+    ],
+    name: '重庆大学本科教学信息网通知',
```

Comment:
Do not repeat namespace name 
```suggestion
    name: '本科教学信息网通知',
```
in route name